webhook_server/tests/test_runner_handler.py

```
@@ -17,24 +39,33 @@ def mock_github_webhook(self) -> Mock:
         mock_webhook.logger = Mock()
         mock_webhook.log_prefix = "[TEST]"
         mock_webhook.repository = Mock()
+        mock_webhook.repository.full_name = "test-owner/test-repo"
         mock_webhook.repository.clone_url = "https://github.com/test/repo.git"
         mock_webhook.repository.owner.login = "test-owner"
         mock_webhook.repository.owner.email = "test@example.com"
-        mock_webhook.token = "test-token"
-        mock_webhook.clone_repo_dir = "/tmp/test-repo"
+        mock_webhook.token = "test-token"  # pragma: allowlist secret
```

Ruff (0.14.1)

Possible hardcoded password assigned to: "token"

(S105)